packages/clerk-js/src/core/resources/AuthConfig.ts

```diff
@@ -14,6 +15,7 @@ export class AuthConfig extends BaseResource implements AuthConfigResource {
   protected fromJSON(data: AuthConfigJSON | null): this {
     this.singleSessionMode = data ? data.single_session_mode : true;
     this.cookielessDev = data ? data.cookieless_dev : false;
+    this.urlBasedSessionSyncing = data ? data.url_based_session_syncing : false;
```

Comment:
( I know that this PR is aims to phase out the `cookielessDev` terminology, so not everything will be removed at once.  )
❓ What is the criteria for keeping here both `this.cookielessDev` and `this.urlBasedSessionSyncing` here and in the files seen bellow ?

Reply:
Well the rationale was:

* Add `urlBasedSessionSyncing`
* When the apps switch over to it, then remove `cookielessDev`

That being said, our dashboard doesn't even use the types here, so they could be removed in this PR as well.
Will update accordingly.